apps/core/scenes/passengerForm/BaggageBundles/BaggageTypeDetail.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { StyleSheet, Icon } from '@kiwicom/universal-components';
+import type { IconNameType } from '@kiwicom/universal-components';
+import { Text } from '@kiwicom/margarita-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { bags, type BagTitleType } from './BagTypes';
+
+type Props = {|
+  +type: BagTitleType,
+  +dimensions: string,
+  +quantity: number,
+  +chosen: boolean,
+  +weight: string,
+|};
+
+function getIcon(type, quantity): IconNameType {
+  if (quantity === 0) {
+    if (type === bags.PERSONAL_ITEM) return 'baggage-personal-item'; // @TODO add "no personal item icon"
```

Comment:
You can find it under name `BaggagePersonalItemNone`, so something like `baggage-personal-item-none` should work.

Reply:
I was trying several things but still not working

Reply:
Oh, I see it is included in Orbit components but somehow missing from our Icon implementation. So todo is ok for now and I'll create an issue for missing icon.